### PR TITLE
renamed manage users to admin tools and implemented a manage users ta…

### PR DESCRIPTION
**Pull request by reesharper**
…b that gives a list of all users as well as functioning edit and delete buttons

### CODE REVIEW

**Review by c-melchor — APPROVED** [review]
There's a console log on line 105 in EditUser, otherwise looks awesome! Nice job, Rees.

File: src/components/pages/AdminTools/ManageUsers.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  const deleteUser = profile => {
+    axios
+      .delete(`${process.env.REACT_APP_API_URI}/profiles/${profile.id}`, {
+        headers: {
+          Authorization: 'Bearer ' + authState.idToken.idToken,
+        },
+      })
+      .then(res => {
+        alert(`${profile.name}'s profile was deleted`);
+        console.log(res.data);
```
Comment (ndossett) [inline src/components/pages/AdminTools/ManageUsers.js:59]: Do we need the console log here?

**Review by ndossett — APPROVED** [review]
Looks nice and clean, just left one comment for you about a console log. This will greatly improve functionality for the admin user.

**Review by ruizaj13 — APPROVED** [review]
Awesome functionality being added, other than what was mentioned above, it looks all good to me!

**Review by crystal-csete — APPROVED** [review]
This is important for the stakeholder to review and manage users.

**Review by JusticeMatthew — APPROVED** [review]
It looks good to me other than just making sure we are properly handling errors so the users know when something breaks, which will inevitably happen!

**Pull request merged**